docs/fargate.md

```diff
@@ -115,6 +124,153 @@ fi
 
 `EFS_ID` is going to be used in setting up Volumes for metric pods.
 
+## Logs
+
+The following are some of the steps needed to setup and enable logs collection on Fargate
+
+### Fluent-bit log router
+
+#### Prerequisites
+
+- An existing Fargate profile that specifies an existing Kubernetes namespace that you deploy Fargate pods to. For more information, see
+  [Create a Fargate profile for your cluster](https://docs.aws.amazon.com/eks/latest/userguide/fargate-getting-started.html#fargate-gs-create-profile)
+
+- An existing
+  [Fargate pod execution role](https://docs.aws.amazon.com/eks/latest/userguide/fargate-getting-started.html#fargate-sg-pod-execution-role)
+
+#### Configuration
+
+Amazon EKS on Fargate offers a built-in log router based on Fluent Bit. This means that you don't explicitly run a Fluent Bit container as a
+sidecar, but Amazon runs it for you. All that you have to do is configure the log router.
+
+Below are the steps to configure the log router on fargate
+
+- Create a dedicated Kubernetes namespace named `aws-observability`
+
+```yaml
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: aws-observability
+  labels:
+    aws-observability: enabled
+```
+
+The fargate log router manages the `Service` and `Input` sections. One needs to create a `ConfigMap` to enable log routing to cloudwatch, an
+example is shown below:
+
+```yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: aws-logging
+  namespace: aws-observability
+data:
+  flb_log_cw: "true" # Set to true to ship Fluent Bit process logs to CloudWatch.
+  output.conf: |
+    [OUTPUT]
+        Name cloudwatch_logs
+        Match   kube.*
+        region region-code
+        log_group_name my-logs
+        log_stream_prefix from-fluent-bit-
+        log_retention_days 60
+        auto_create_group true
+```
+
+You can stream logs from Fargate directly to Amazon CloudWatch, using the output plugin shown above. Cloudwatch is currently the only
+supported output plugin in fluent-bit running on EKS fargate. Note that the log router setup depends on the prerequisites. Otherwise, one
```

Comment:
```suggestion
supported output plugin in Fluent Bit running on EKS fargate. Note that the log router setup depends on the prerequisites. Otherwise, one
```